schema/function.go

```diff
@@ -198,23 +216,36 @@ func (f FunctionSchema) Display() Display {
 
 func (f FunctionSchema) String() string {
 	result := f.ID() + "(" + strings.Join(f.ParameterTypeNames(), ", ") + ") "
-	if f.OutputValue != nil {
-		result += string(f.OutputValue.TypeID())
-	} else {
-		result += "void"
-	}
+	result += getReturnTypeString(f.OutputValue, false)
 	return result
 }
 
+func getReturnTypeString(returnType Type, hasError bool) string {
+	switch {
+	case returnType != nil:
+		if hasError {
+			return "(" + string(returnType.TypeID()) + ", error)"
+		} else {
+			return string(returnType.TypeID())
+		}
+	case hasError:
+		return "error"
+	default:
+		return "void"
+	}
```

Comment:
Again, only if you are making changes for some other reason...this might be more graceful code:
```golang
	switch {
	case returnType != nil && hasError:
		return "(" + string(returnType.TypeID()) + ", error)"
	case returnType != nil:
		return string(returnType.TypeID())
	case hasError:
		return "error"
	default:
		return "void"
	}
```

Reply:
I'll make the change.